kit/src/main/java/com/oracle/javafx/scenebuilder/kit/skeleton/SkeletonWindowController.java

```diff
@@ -120,6 +126,28 @@ public SkeletonWindowController(EditorController editorController, String docume
         if (editorController.getFxomDocument() != null) {
             editorController.getFxomDocument().sceneGraphRevisionProperty().addListener(fxomDocumentRevisionListener);
         }
+        
+        this.copyAccelerator = getCopyToClipboardKeyAccelerator();
+    }
+
+    private KeyCharacterCombination getCopyToClipboardKeyAccelerator() {
+        return new KeyCharacterCombination("c", KeyCombination.SHORTCUT_DOWN);
+    }
+
+    private EventHandler<KeyEvent> getCopyKeyEventHandler() {
+        return event -> handleCopyToClipboardEvent(event, "c".equals(event.getText()), event.isShortcutDown());
+    };
```

Comment:
@abhinayagarwal Same here, if it works on Mac, there wont be any need to have this method. Works already fine on Windows 10. Will test on Mac soon.